### PR TITLE
Issue 443/search language

### PR DESCRIPTION
**Pull request by PhotoNomad0**
Added method to search manifest table.

Issues: https://github.com/unfoldingWord-dev/door43.org/issues/443 and https://github.com/unfoldingWord-dev/door43.org/issues/484

### CODE REVIEW

**Comment by PhotoNomad0** [comment]
Adding support for https://github.com/unfoldingWord-dev/door43.org/issues/484.

Still in Process.

File: README.md [hunk 1]
```diff
@@ -50,7 +57,7 @@ The master branch is available at https://live.door43.org (soon to be at https:/
 
 #### Pre Production Testing
 
-You may run `make test`, or `make build`, or `make serve` to test and review your changes locally.  Once the `cibuild.sh` script passes successfully locally, you may commit and push to the `develop` branch.  You can do this by running `make commit`.
+You may run `make test`, or `make build`, or `make serve` to test and review your changes locally.  Once the `cibuild.sh` script passes successfully locally, push `test` branch.  Follow setup instructions in `s3_test_push.sh` and then run the script  `s3_test_push.sh`.
```
Comment (richmahn) [inline README.md]: Don't we still need people to be making a PR to the develop branch once everything is working locally and on 'test'? Still need the line about committing and pushing (making a PR) to the develop branch. Or does `make commit` not do that?

File: js/i18n.js [hunk 2]
```diff
@@ -345,16 +345,41 @@ function getLanguageListItems($textBox, callback) {
...
+        }
+    );
+}
 
 $().ready(function () {
   window.setTimeout(searchForResources(''), 300);
   window.setTimeout(setupLanguageSelector(), 500);
 
   $('#search-td').on('click', function (){
     var search_for = document.getElementById('search-for').value;
```
Comment (richmahn) [inline js/i18n.js]: While you didn't write this line, can we get all code being jQuery if that is what we are using? Should be `var search_for = $('#search-for').val();`

File: README.md (hunk 1 above)
Comment (PhotoNomad0) [inline README.md]: @richmahn - Not sure what you are asking here, but this section is titled pre-production testing.  As I understand we are now doing that on `test`.  And pushing to that is now done by script.

Are you wanting a note added that after they make a PR against develop branch and it has been merged, that Travis will automatically deploy to dev for further testing?

Comment (PhotoNomad0) [inline README.md]: Maybe add a section `Push to Develop`?

Comment (PhotoNomad0) [inline README.md]: PS - `make commit` makes me nervous.  I don't like the thought of a script committing changes automatically.  I prefer to review the changes.

**Comment by PhotoNomad0** [comment]
@richmahn - fixed UI to do language plus full text search.  Results still shown as alert.

**Pull request merged**